data/geany-3.0.css

```diff
@@ -1,10 +0,0 @@
-@import "geany.css";
-
-/* make close button on the editor's tabs smaller */
-#geany-close-tab-button {
-	-GtkWidget-focus-padding: 0;
-	-GtkWidget-focus-line-width: 0;
-	-GtkButton-default-border: 0;
-	-GtkButton-default-outside-border: 0;
-	-GtkButton-inner-border: 0;
-}
```

Comment:
> Okay apparently GTK 3.22 doesn't complain.

My bad, I mistested.  GTK 3.22 complains a lot:
```console
(geany:20553): Gtk-WARNING **: Theme parsing error: geany.css:57:27: The style property GtkWidget:focus-padding is deprecated and shouldn't be used anymore. It will be removed in a future version

(geany:20553): Gtk-WARNING **: Theme parsing error: geany.css:58:30: The style property GtkWidget:focus-line-width is deprecated and shouldn't be used anymore. It will be removed in a future version

(geany:20553): Gtk-WARNING **: Theme parsing error: geany.css:59:28: The style property GtkButton:default-border is deprecated and shouldn't be used anymore. It will be removed in a future version

(geany:20553): Gtk-WARNING **: Theme parsing error: geany.css:60:36: The style property GtkButton:default-outside-border is deprecated and shouldn't be used anymore. It will be removed in a future version

(geany:20553): Gtk-WARNING **: Theme parsing error: geany.css:61:26: The style property GtkButton:inner-border is deprecated and shouldn't be used anymore. It will be removed in a future version
```